crates/abq_cli/src/abq_config.rs

```diff
@@ -0,0 +1,121 @@
+use anyhow::anyhow;
+use std::{fs, path::PathBuf};
+
+use abq_hosted::AccessToken;
+use etcetera::{app_strategy, AppStrategy, AppStrategyArgs};
+use serde_derive::{Deserialize, Serialize};
+
+#[derive(Deserialize, Serialize, Debug, PartialEq, Eq)]
+pub struct AbqConfig {
+    pub rwx_access_token: AccessToken,
+}
+
+pub enum AbqConfigFileMode {
+    // Conventional unix location (e.g ~/.config/abq/config.toml)
+    Conventional,
```

Comment:
Just a note, if you want these comment to be documentation comments (show up when you hover over a reference, for example), use `///` rather than `//`.